pkg/controller/daemon/daemon_controller.go

```diff
@@ -1327,11 +1342,48 @@ func NewPod(ds *extensions.DaemonSet, nodeName string) *v1.Pod {
 	return newPod
 }
 
+// nodeSelectPredicates are the predicates that used to select node candidates for DaemonSet
```

Comment:
Please change to:
nodeSelectionPredicates runs a set of predicates that select candidate nodes for the DaemonSet.